projects/json-pointer-helpers/src/json-pointers/json-pointer-helpers.ts

```diff
@@ -57,15 +57,21 @@ function relative(pointer: string, from: string) {
   return compile(targetDecoded.slice(fromDecoded.length));
 }
 
-function startsWith(pointer: string, pattern: string[]): boolean {
+function startsWith(
+  pointer: string,
+  pattern: string[],
+  exact: boolean = false
```

Comment:
Can we change this to be an `options` arg with key exact so it's easier to read when it's used downstream

Reply:
Done